crates/turborepo/tests/common/mod.rs

```diff
@@ -0,0 +1,56 @@
+use std::{path::Path, process::Command};
+
+use camino::Utf8Path;
+
+pub fn setup_fixture(
+    fixture: &str,
+    package_manager: Option<&str>,
```

Comment:
suggestion: it would be nice to keep this explicit so it's really clear from looking at the test what it's gonna run with.  or, if not, it'd be nice if there was some way to make `check!` variadic (is there?).